database/factories/TimeLogFactory.php

```diff
@@ -25,19 +25,45 @@
 $factory->define(TimeLog::class, function (Faker $faker) {
 
     $uuid = new UUID('log');
-    $user = User::where('id', 1)->first();
+    $user = User::all()->random();
     $orgId = $user->organization_id;
 
     $timeSheet = TimeSheet::where('organization_id', $orgId)->first();
 
-    $date =  "2019-02-01";
-    $time = "06:30:44";
+
+    // Generate a random date with ranges
+    // upper-limit -> today, lower-limit -> today-6months
+    $tempDate = Carbon::now()->subMonths(rand(0,6))->subDays(rand(0,7))->subHours(rand(0, 12));
+
+    $year = $tempDate->year;
```

Comment:
if the tempdate object has the same property names as the ones we need why we doing this?

Reply:
I think I could rename it to `$date`